cmd/user-cluster-webhook/main.go

```diff
@@ -101,28 +99,35 @@ func main() {
 		log.Fatalw("Failed to create the user cluster manager", zap.Error(err))
 	}
 
-	// apply the CLI flags for configuring the  webhook server to the manager
-	if err := options.webhook.Configure(seedMgr.GetWebhookServer()); err != nil {
+	// Apply the CLI flags for configuring the webhook servers.
+	if err := options.seedWebhook.Configure(seedMgr.GetWebhookServer()); err != nil {
+		log.Fatalw("Failed to configure webhook server", zap.Error(err))
+	}
+	if err := options.userWebhook.Configure(userMgr.GetWebhookServer()); err != nil {
 		log.Fatalw("Failed to configure webhook server", zap.Error(err))
 	}
 
 	// add APIs we use
 	addAPIs(seedMgr.GetScheme(), log)
+	addAPIs(userMgr.GetScheme(), log)
 
 	// /////////////////////////////////////////
 	// add pprof runnable, which will start a websever if configured
 
 	if err := seedMgr.Add(&options.pprof); err != nil {
-		log.Fatalw("Failed to add the pprof handler", zap.Error(err))
+		log.Fatalw("Failed to add the pprof handler (seed manager)", zap.Error(err))
+	}
+	if err := userMgr.Add(&options.pprof); err != nil {
```

Comment:
Does this make sense? We use the manager as a simple way to start the pprof stuff, but even with many managers there is no need to have multiple pprof things running in the same Go process, as they have no relation to the manager that started them.

Reply:
I though that relation between pprof things and managers has to be 1 to 1, but it makes sense to remove one from what you've explained.